integration/pages/BaseTestPage.js

```diff
@@ -0,0 +1,66 @@
+const BasePage = require('./BasePage');
+const assert = require('assert');
+const { By } = require('selenium-webdriver');
+
+const ELEMENTS = {
+  appHeader: By.css('[data-testid=\'app-name\']'),
+  meetingIdInputBox: By.css('[data-testid=\'input\'][name=\'Meeting Id\']'),
+  nameInputBox: By.css('[data-testid=\'input\'][name=\'Name\']'),
+  joinMeetingButton: By.css('[data-testid=\'button\'][label=\'Join Meeting\']'),
+  leaveMeetingButton: By.css('[data-testid=\'button\'][label=\'Leave\']'),
+  meetingStatus: By.id('meeting-status'),
+};
+
+class BaseTestPage extends BasePage {
+  constructor(driver) {
+    super(driver);
+  }
+
+  async load(route) {
```

Comment:
Can one use `visit` directly over `load`? This looks like an un-necessary extra method on the extended class.

Reply:
Removed `load()` and use `visit()` instead.